jetson/nav/search/searchStateMachine.cpp

```diff
@@ -284,28 +285,86 @@ NavState SearchStateMachine::executeSearchDrive( Rover* phoebe, const rapidjson:
 } // executeSearchDrive()
 
 // Executes the logic for turning to the target.
+// Starts by turnign the gimbal towards the target. Then holds for 200 cycles to confirm aquisition
 // If the rover loses the target, will continue to turn using last known angles.
+// Once target location confirmed, rover turns towards the target, then rights its gimbal
 // If the rover finishes turning to the target, it goes into waiting state to
 // give CV time to relocate the target
 // Else the rover continues to turn to to the target.
-NavState SearchStateMachine::executeTurnToTarget( Rover* phoebe )
+NavState SearchStateMachine::executeTurnToTarget( Rover* phoebe, const rapidjson::Document& roverConfig  )
 {
-    if( phoebe->roverStatus().leftTarget().distance < 0 )
+    static int turn_to_target_state = 0;
```

Comment:
change name to stage instead of state. also, use camelCase throughout instead of snake case